targets/curie_bsp/jerry_app/quark/main.c

```diff
@@ -51,17 +52,17 @@ void jerry_resolve_error (jerry_value_t ret_value)
     jerry_char_t *err_str_buf = (jerry_char_t *) balloc (err_str_size, NULL);
     jerry_size_t sz = jerry_string_to_char_buffer (err_str_val, err_str_buf, err_str_size);
     err_str_buf[sz] = 0;
-    jerry_port_console ("Script Error: unhandled exception: %s\n", err_str_buf);
+    printk ("Script Error: unhandled exception: %s\n", err_str_buf);
     bfree(err_str_buf);
     jerry_release_value (err_str_val);
   }
 }
 
 void help ()
 {
-  jerry_port_console ("Usage:\n");
-  jerry_port_console ("js e 'JavaScript Command'\n");
-  jerry_port_console ("eg. js e print ('Hello World');\n");
+  printk ("Usage:\n");
+  printk ("js e 'JavaScript Command'\n");
+  printk ("eg. js e print ('Hello World');\n");
```

Comment:
In the curie-bsp port, line141 in this file
```
   jerry_value_t print_func_name_val = jerry_create_string ((jerry_char_t *) "print");
   print_function = jerry_get_property (global_obj_val, print_func_name_val);
```
We refer the main-unix and get the print function from global. Then call the print on every ret of repl.
So we need to still paste the new implementation of `print_handler` here and `register_js_function`

Do you want to do it in this patch? or I can post a following one after it lands